publish2rs.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+import os
+import kinto_http
+import hashlib
+import pathlib
+import uuid
+
+from settings import config
+
+from constants import WEBKIT_LISTS_DIR
+
+def get_config_if_env(env_var, config_section, config_option, fallback=""):
+    """
+    Return the config value if the environment variable exists; otherwise, return the fallback.
+    """
+    if env_var in os.environ:
+        return config.get(config_section, config_option, fallback=fallback)
+    return fallback
+
+def get_file_hash(list_path):
+    with open(list_path, "rb") as f:
+        return hashlib.sha256(f.read()).hexdigest()
+
+SERVER = get_config_if_env("SERVER", "main", "remote_settings_url")
+BUCKET = config.get("main", "remote_settings_bucket")
```

Comment:
I think you can hardcode `main-workspace` here, becuase I don't see any use-case where it would be different